src/Faker/Provider/Miscellaneous.php

```diff
@@ -273,10 +300,21 @@ public static function languageCode()
 
     /**
      * @example 'EUR'
+     *
      * @link https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
      */
     public static function currencyCode()
     {
         return static::randomElement(static::$currencyCode);
     }
+
+    /**
+     * Returns an encoded Unicode Character between U+1F600 and U+1F637.
+     *
+     * @link https://en.wikipedia.org/wiki/Emoji#Unicode_blocks
+     */
+    public static function emoji()
+    {
+        return json_decode('"' . static::randomElement(static::$emoji) . '"');
```

Comment:
why use `json_decode` for that? Why not store unicode directly in the `static::$emoji` source? That would be faster.